app/views/wpcc/your_details/new.html.erb

```diff
@@ -2,109 +2,131 @@
             <div class="callout">
               <p><%= t('wpcc.details.callout__lt16') %></p>
             </div>
           </div>
-
           <div class="form__row details__callout details__callout--inactive" data-dough-callout-optIn>
             <div class="callout">
               <p><%= t('wpcc.details.callout__optIn') %></p>
             </div>
           </div>
-
           <div class="form__row details__callout details__callout--inactive" data-dough-callout-gt74>
             <div class="callout">
               <p><%= t('wpcc.details.callout__gt74') %></p>
             </div>
           </div>
         </div>
       </div>
-
-      <div class="details__row">
+      <div class="details__row" data-dough-component="PopupTip">
         <div class="details__field">
           <div class="form__row details__salary">
+            <div class="details__field-label">
+              <%= f.label :salary, t('wpcc.details.salary.label'), class: 'form__label-heading' %>
+              <button type="button" class="popup-tip__button" data-dough-popup-trigger>
+                <span aria-hidden="true">i</span>
+                <span class="visually-hidden"><%= t('wpcc.details.tooltip_show') %></span>
+              </button>
+            </div>
             <div class="details__salary-amount">
-              <%= f.label :salary, t('wpcc.details.salary.label'), class: 'form__label-heading details__field-label' %>
               <%= f.text_field :salary %>
             </div>
             <div class="details__salary-frequency">
-              <%= f.label :salary_frequency, t('wpcc.details.salary.frequency.label'), class: 'form__label-heading details__field-label' %>
+              <%= f.label :salary_frequency, t('wpcc.details.salary.frequency.label'), class: 'form__label-heading details__field-label visually-hidden' %>
               <%= f.select(:salary_frequency, options_for_select(@your_details_form.salary_frequency_options, @your_details_form.salary_frequency)) %>
             </div>
           </div>
         </div>
-
-        <div class="details__helper">
-          <div class="field-help-text">
-            <div class="tooltip__content-container">
-              <p><%= t('wpcc.details.salary.tooltip') %></p>
-            </div>
-          </div>
+        <div data-dough-popup-content class="popup-tip__content details__helper">
+          <p><%= t('wpcc.details.salary.tooltip') %></p>
+          <button data-dough-popup-close type="button" class="popup-tip__close">
+            <span aria-hidden="true">
+              <svg xmlns="http://www.w3.org/2000/svg" class="svg-icon svg-icon--mobile-close-box">
+                <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#svg-icon--mobile-close-box"></use>
+              </svg>
+            </span>
+            <span class="visually-hidden">
+              <%= t('wpcc.details.tooltip_hide') %>
+            </span>
+          </button>
```

Comment:
This html section is identical to the other. What do you think to move to a partial, passing to the partial the tooltip message?